### PR TITLE
Update pullspec in openshift-oadp-operator-master.yaml

### PR DESCRIPTION
**Pull request by deepakraj1997**

### CODE REVIEW

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ operator:
   bundles:
   - dockerfile_path: build/Dockerfile.bundle
   substitutions:
-  - pullspec: quay.io/konveyor/oadp-operator.v0.2.3:latest
+  - pullspec: quay.io/konveyor/oadp-operator.v0.2.6:latest
```
Comment (dymurray) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml:22]: `quay.io/konveyor/oadp-operator:latest`

Comment (deepakraj1997) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml]: Changed w.r.t this: https://github.com/openshift/oadp-operator/pull/150/files (Pasting for reference)

Comment (dymurray) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml]: LGTM

**Review by dymurray — APPROVED** [review]

**Pull request merged**